### ISSUE TITLE
Révisions en Français: English content on French page.

### ISSUE THREAD

**Issue by qanitah** [mmiron-HC/WST-RSAMS#86]
URL: https://hcdev.openplus.ca/fr/node/2526/revisions 
User Agent: Chrome, Firefox
Language: French 

Issue Description: There is English text on French page. All text should be on the language of the page.

Steps to reproduce: Compare selected revisions is already in English but the English error message appear when user clicks "Compare selected revisions". 

Expected results: All text should be on the language of the page.

![image](https://user-images.githubusercontent.com/14280062/129383124-e9cdb84a-9646-4301-9379-254d27e95060.png)

![image](https://user-images.githubusercontent.com/14280062/129396271-19acff3f-dd57-4ecf-af13-39f097c2e733.png)

**Comment by echouin** [comment]
OP has confirmed this is completed.

**Comment by qanitah** [comment]
Some is fixed but some still in English
![image](https://user-images.githubusercontent.com/14280062/136257059-42d950c1-c4b3-4907-b021-ca4d27f53e43.png)

**Comment by davidgotlieb** [comment]
Fixed on live site

**Comment by ndolter** [comment]
@davidgotlieb Can you please confirm that since the radio buttons can no longer be blank, that the text in the 3 screenshots above is no longer possible to view?  If so, I will close this ticket.  There are some new language of parts issues that I noticed on the French page, but I will address them in a new ticket.

**Comment by davidgotlieb** [comment]
Hi @ndolter I don't see how you could possibly get that error anymore. As you point out  both radio buttons are pre selected so there's no way to have it blank.

All other content on the page is now in French so I think this is fully resolved.

**Comment by ndolter** [comment]
Hi @davidgotlieb, thanks.  This is the language issue that I'm seeing on the page and will create a new ticket to address:
![image](https://user-images.githubusercontent.com/68117281/160117295-60c105cc-c53c-4fae-8084-c126a377fd47.png).

Closing this ticket as the issue has been resolved.
